xml/Microsoft.Build.Globbing/MSBuildGlobWithGaps.xml

```diff
@@ -146,10 +146,23 @@
         <Parameter Name="stringToMatch" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="stringToMatch">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="stringToMatch">The string to match. If the string represents a relative path, it will get normalized against the glob's root. Cannot be <see langword="null" />.</param>
+        <summary>Matches the given <paramref name="stringToMatch" /> against the glob.</summary>
+        <returns><see langword="false" /> if <paramref name="stringToMatch" /> contains invalid path or file characters; <see langword="true" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns>
            <see langword="false" /> if <paramref name="stringToMatch" /> contains invalid path or file characters; <see langword="true" /> otherwise.</returns>
```